tests/integration/test_xrdsst.py

```diff
@@ -123,6 +123,13 @@ def step_enable_service_description(self, client_id):
             service_description = get_service_description(self.config, client_id)
             assert service_description["disabled"] is False
 
+    def step_add_service_access_rights(self):
+        with XRDSSTTest() as app:
+            service_controller = ServiceController()
+            service_controller.app = app
+            service_controller.load_config = (lambda: self.config)
+            service_controller.add_rights
```

Comment:
Added a check whether service_client has been created, but on my computer this fails because it seems register_client gets HTTP500. Please try if you get the same result. In E2E tests it works at least. It seems that for some reason client registration does not succeed in integration tests and that is the reason why service_client is not added, because it is the same actual client and it need to be registered first.

Reply:
```
    def step_add_service_access(self, client_id):
        with XRDSSTTest() as app:
            service_controller = ServiceController()
            service_controller.app = app
            service_controller.load_config = (lambda: self.config)
            service_controller.add_access
```
Integration test fails, the ^^ access invocation is still not done here!

Reply:
So after access invocation was added, the integration test seems to pass on my local dev machine.